File: Components/Confirm.js
import React from 'react';
import { View, Text, StyleSheet } from 'react-native';
import BackgroundWhite from '../Widgets/BackgroundWhite';
import Logo from '../Widgets/Logo';
import BottomNavbar from '../Widgets/BottomNavbar';
import { Ionicons } from '@expo/vector-icons'; // For the favorite heart icon

export default function Confirm() {
  return (
    <BackgroundWhite>
      <View style={styles.safeArea}>
        {/* Top spacing */}
        <View style={{ height: 40 }} />

        {/* Logo */}
        <Logo width={180} />

        {/* Spacing */}
        <View style={{ height: 40 }} />

        {/* Main content */}
        <View style={styles.contentContainer}>
          <Ionicons
            name="heart"
            size={80}
            color="rgb(144,66,216)"
          />
          <View style={{ height: 20 }} />
          <Text style={styles.title}>Compte confirmé 💕</Text>
          <View style={{ height: 12 }} />
          <Text style={styles.description}>
            Bienvenue dans votre espace couple.{'\n'}
            Construisez, partagez et grandissez ensemble ✨
          </Text>
        </View>

        {/* Footer navbar */}
        <BottomNavbar />
      </View>
    </BackgroundWhite>
  );
}

const styles = StyleSheet.create({
  safeArea: {
    flex: 1,
    justifyContent: 'space-between',
    alignItems: 'center',
    paddingTop: 0,
    width: '100%',
  },
  contentContainer: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    paddingHorizontal: 32,
  },
  title: {
    fontSize: 24,
    fontWeight: 'bold',
    textAlign: 'center',
  },
  description: {
    fontSize: 16,
    textAlign: 'center',
  },
});

File: lib/pages/confirm.dart
import 'package:flutter/material.dart';
import '../Widgets/Background_white.dart';
import '../Widgets/Logo.dart';
import '../Widgets/Navbar.dart';

class Confirm extends StatelessWidget {
  const Confirm({super.key});

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Background_white(
        child: SafeArea(
          child: Column(
            children: [
              const SizedBox(height: 40),

              // Logo
              Logo(width: 180),

              const SizedBox(height: 40),

              // Main content
              Expanded(
                child: Column(
                  mainAxisAlignment: MainAxisAlignment.center,
                  children: const [
                    Icon(
                      Icons.favorite,
                      color: Color.fromARGB(255, 144, 66, 216),
                      size: 80,
                    ),
                    SizedBox(height: 20),
                    Text(
                      "Compte confirmé 💕",
                      style: TextStyle(
                        fontSize: 24,
                        fontWeight: FontWeight.bold,
                      ),
                    ),
                    SizedBox(height: 12),
                    Padding(
                      padding: EdgeInsets.symmetric(horizontal: 32),
                      child: Text(
                        "Bienvenue dans votre espace couple.\nConstruisez, partagez et grandissez ensemble ✨",
                        textAlign: TextAlign.center,
                        style: TextStyle(fontSize: 16),
                      ),
                    ),
                  ],
                ),
              ),

              // Footer navbar
              const BottomNavbar(),
            ],
          ),
        ),
      ),
    );
  }
}
